src/current/v23.1/molt-verify.md

```diff
@@ -78,19 +78,28 @@ The following databases are currently supported:
 
 Flag | Description
 ----------|------------
-`--concurrency int` | Number of shards to process at a time. <br>Default value: 16 <br>For faster verification, set this flag to a higher value. <br>Note: Table splitting by shard only works for `int`, `uuid`, and `float` data types.
-`--table_splits int` | Number of shards to split the table into. <br>Default value: 16
-`--row_batch_size int` | Number of rows to get from a table at a time. <br>Default value: 20000
+`--source` | (Required) Connection string for the source database.
+`--target` | (Required) Connection string for the target database.
+`--concurrency` | Number of shards to process at a time. <br>**Default:** 16 <br>For faster verification, set this flag to a higher value. <br>Note: Table splitting by shard only works for [`INT`](int.html), [`UUID`](uuid.html), and [`FLOAT`](float.html) data types.
+`--table_splits` | Number of shards to split the table into. <br>**Default:** 16
```

Comment:
these switched to `-` instead. i actually removed `--table-splits` because i don't deem it production ready.

Reply:
fwiw when I just pulled in latest and built it, I'm still seeing two dashes for these flags. Unless this meant something else and I didn't understand

Reply:
@otan can you clarify here?